### ISSUE TITLE
Expo 5.1.1 "minify" must be a boolean if specified error 

### ISSUE THREAD

**Issue by danieldanielecki** [expo/expo-cli#4205]
### Summary

After upgrading from `5.0.1` to `5.1.1`, while trying to run the simulator, it breaks beforehand with mysterious error output of `"minify" must be a boolean if specified error Command failed with exit code 1.`. Downgrading to `5.0.1` makes it work again.

### Environment

```
  Expo CLI 5.1.1 environment info:
    System:
      OS: macOS 12.0.1
      Shell: 5.8 - /bin/zsh
    Binaries:
      Node: 15.14.0 - /usr/local/bin/node
      Yarn: 1.22.10 - /usr/local/bin/yarn
      npm: 7.7.6 - /usr/local/bin/npm
    Managers:
      CocoaPods: 1.11.2 - /usr/local/bin/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 21.2, iOS 15.2, macOS 12.1, tvOS 15.2, watchOS 8.3
      Android SDK:
        API Levels: 30, 32
        Build Tools: 29.0.2, 32.0.0
        System Images: android-32 | Google Play Intel x86 Atom_64
    IDEs:
      Android Studio: 2021.1 AI-211.7628.21.2111.8092744
      Xcode: 13.2.1/13C100 - /usr/bin/xcodebuild
    npmPackages:
      expo: ^44.0.0 => 44.0.5 
      react: 17.0.1 => 17.0.1 
      react-dom: 17.0.1 => 17.0.1 
      react-native: 0.64.3 => 0.64.3 
      react-native-web: 0.17.1 => 0.17.1 
    npmGlobalPackages:
      expo-cli: 5.1.1
    Expo Workflow: bare
```

### Please specify your device/emulator/simulator platform, model and version

iPhone 12, iOS 15.2 & Pixel_2_API_32

### Error output

"minify" must be a boolean if specified error Command failed with exit code 1.

### Reproducible demo or steps to reproduce from a blank project

1. `npm install -g expo-cli`
2. `expo run:ios`

**Comment by brentvatne** [comment]
can you try deleting your `.expo` directory in the project and running `expo run:ios` again?

**Comment by EvanBacon** [comment]
Related https://github.com/expo/expo-cli/pull/4166

**Comment by danieldanielecki** [comment]
Before deleting, on `5.0.3` (`5.0.1` was crashing as well).
```
{
  "hostType": "lan",
  "lanType": "ip",
  "dev": false,
  "minify": true,
  "urlRandomness": "blabla",
  "https": false,
  "devClient": true,
  "scheme": "com.xyz.app"
}

```
After deleting and `expo run:ios`, on `5.1.2`
```
{
  "scheme": "com.xyz.app",
  "hostType": "lan",
  "lanType": "ip",
  "devClient": true,
  "dev": true,
  "minify": false,
  "urlRandomness": null,
  "https": false
}
```

Problem solved, thank you!

**Comment by tamis-laan** [comment]
Same problem, even when removing `.expo`, `.expo-shared` and `node_modules`:

``` console
XDLError: "minify" must be a boolean if specified
    at assertValidOptions (/.../.config/yarn/global/node_modules/xdl/src/UrlUtils.ts:288:11)
    at ensureOptionsAsync (/.../.config/yarn/global/node_modules/xdl/src/UrlUtils.ts:334:10)
    at constructUrlAsync (/.../.config/yarn/global/node_modules/xdl/src/UrlUtils.ts:380:16)
    at Object.constructDebuggerHostAsync (/.../.config/yarn/global/node_modules/xdl/src/UrlUtils.ts:174:10)
    at async Promise.all (index 0)
    at getExpoGoConfig (/.../.config/yarn/global/node_modules/xdl/src/start/ManifestHandler.ts:213:34)
    at getManifestResponseAsync (/.../.config/yarn/global/node_modules/xdl/src/start/ManifestHandler.ts:268:24)
    at /.../.config/yarn/global/node_modules/xdl/src/start/ManifestHandler.ts:151:49
```

**Comment by brentvatne** [comment]
fixed in expo-cli@5.1.3

sorry for the trouble @tamis-laan and @thelamedia. you can install this version with `npm i -g expo-cli@5.1.3`

**Comment by thelamedia** [comment]
Confirming the fix.  Thanks!
